doc/manual/src/architecture/architecture.md

```diff
@@ -0,0 +1,78 @@
+# Architecture
+
+This chapter describes how Nix works.
+It should help users understand why Nix behaves as it does, and it should help developers understand how to modify Nix and how to write similar tools.
+
+## Overview
+
+Nix consists of [hierarchical layers][layer-architecture].
+
+```
++-----------------------------------------------------------------+
+| Nix                                                             |
```

Comment:
```suggestion
| packaging system                                                |
```

Tbh this feels a bit absurd, but where do we draw the line of what terminology is "too specific" for the introduction?

By the same reasoning, we should perhaps replace `evaluates to` by `computes`?

Reply:
> Tbh this feels a bit absurd, but where do we draw the line of what terminology is "too specific" for the introduction?

Exactly, this is why it is that way and nothing else.

> By the same reasoning, we should perhaps replace evaluates to by computes?

We use "evaluate" because that is what pure languages do. That term is also understood by people not into programming language theory, as it is self-explanatory - evaluation produces a value.